app/models/service_area.rb

```diff
@@ -3,6 +3,8 @@ class ServiceArea < ApplicationRecord
   translates :name
   translates :description, type: :text
 
+  after_initialize :create_location_if_not_exists
```

Comment:
@h-m-m so i think we def want to ditch this commit. lmk if you want me to push a change doing that, or if it'll get rolled into the rebase